app/src/main/cpp/HandModels.cpp

```diff
@@ -265,11 +271,13 @@ HandModels::UpdateHandModel(const Controller& aController) {
 
   glBindBuffer(GL_ELEMENT_ARRAY_BUFFER, 0);
   glBindBuffer(GL_ARRAY_BUFFER, 0);
+
+  // Since this is called lazily at render time, a flush may be needed with some drivers
+  glFlush();
```

Comment:
Why do we need this? AFAIK we're using double buffering so this in practice should make no differentce, shouldn't it?

Reply:
Same answer as above.